tests/test_clock.py

```diff
@@ -26,9 +26,8 @@ def test_scheduler_has_job(self):
     def test_clock_expects_config_to_be_ready(self):
         assert not get_config().ready
         jobs = self.clock.scheduler.get_jobs()
-        with raises(RuntimeError) as excinfo:
+        with raises(RuntimeError, message='Config loading not finished'):
```

Comment:
I'd been the same. I thought it was a bit icky when I first heard the idea, but then when I worked on a project that used it the usability really outweighed any downsides.